.travis.yml

```diff
@@ -27,6 +27,6 @@ install:
 # command to run tests
 script:
   - flake8 . --exclude=.venv,.build --ignore=E501,F403,E402
-  - nosetests --with-doctest deeptools/test `ls deeptools/*.py | grep -v correct`
+  - nosetests --with-doctest deeptools/test deeptools --ignore-files="correctGCBias"
```

Comment:
More or less, though `deeptools/test` should be `deeptools.test`. It runs 66 tests, at least. I'll figure out what the missing 3 are and add them manually.